src/calcligatureorneumeposfunctor.cpp

```diff
@@ -347,17 +347,7 @@ FunctorCode CalcLigatureOrNeumePosFunctor::VisitNeume(Neume *neume)
             }
         }
 
-        // If the nc overlaps with the previous, move it back from a line width
-        if (overlapWithPrevious) {
-            xRel -= lineWidth;
-        }
-
         nc->SetDrawingXRel(xRel);
-        // The first glyph set the spacing - unless we are starting a ligature, in which case no spacing should be added
-        // between the two nc
-        if (!previousLig) {
-            xRel += m_doc->GetGlyphWidth(nc->m_drawingGlyphs.at(0).m_fontNo, staffSize, false);
-        }
```

Comment:
I think what you want is not do this when there is a facsimile. It should not be removed because it is use to calculate the position of the neumes when there is no facsimile